iOS-remote-node-setup-docker.md

```diff
@@ -0,0 +1,116 @@
+# Zap iOS: Remote Node setup with Docker
+
+Zap iOS will eventually ship with on-device LND support, but for now you have to connect to a remote [LND](https://github.com/lightningnetwork/lnd) node.
+
+This tutorial covers running a "zap-ready" LND node inside a [Docker container](https://www.docker.com/resources/what-container) and connecting to it from Zap.
+
+## [Requirement] Server with docker installed
+
+This tutorial will **not** cover how to set up a Digital Ocean Droplet (or some other cloud server). Install instruction for docker can be be found [here](https://docs.docker.com/install/). A quick way to spin up one is to use Digital Oceans [One-click install and deploy Docker](https://www.digitalocean.com/products/one-click-apps/docker/).
+
+## Start the LND node
+
+The great thing about Docker is that you don't have to worry about installing all the required software. We have already built an image for you to use, so let's just fire it up 🚀
+
+```
+docker run --name lnd -p 10009:10009 -v $(pwd)/lnd-data:/root/.lnd -d lnzap/lnd:latest lnd --bitcoin.active --bitcoin.testnet --debuglevel=info --bitcoin.node=neutrino --neutrino.connect=testnet1-btcd.zaphq.io --neutrino.connect=testnet2-btcd.zaphq.io --autopilot.active --rpclisten=0.0.0.0:10009
```

Comment:
Additionally, you shouldn't override the default entrypoint as this will bypass `gosu` and cause the daemon to run as the root user which is undesirable. You should remove `lnd` from after the image name.

The final command should be:

```
docker run --name lnd -p 10009:10009 -v $(pwd)/lnd-data:/root/.lnd lnzap/lnd:latest \
  --bitcoin.active \
  --bitcoin.testnet \
  --debuglevel=info \
  --bitcoin.node=neutrino \
  --neutrino.connect=testnet1-btcd.zaphq.io \
  --autopilot.active --rpclisten=0.0.0.0:10009
```

See https://github.com/LN-Zap/docker-lnd/blob/master/docker-entrypoint.sh for how/why.